aqua/MenuMeters/files/patch-MenuMeters.xcodeproj-project.pbxproj.diff

```diff
@@ -1,18 +1,10 @@
---- MenuMeters.xcodeproj/project.pbxproj.orig	2011-07-02 23:27:50.000000000 -0500
-+++ MenuMeters.xcodeproj/project.pbxproj	2011-11-13 01:18:18.000000000 -0600
-@@ -1643,7 +1643,6 @@
- 				SEPARATE_STRIP = YES;
- 				WARNING_CFLAGS = (
- 					"-Wall",
--					"-Werror",
- 					"-Wshorten-64-to-32",
- 				);
- 			};
-@@ -1674,7 +1673,6 @@
- 				SEPARATE_STRIP = YES;
- 				WARNING_CFLAGS = (
- 					"-Wall",
--					"-Werror",
- 					"-Wshorten-64-to-32",
- 				);
+--- MenuMeters.xcodeproj/project.pbxproj.orig
++++ MenuMeters.xcodeproj/project.pbxproj
+@@ -1380,6 +1380,7 @@
+ 				PRODUCT_NAME = MenuMeters;
+ 				SDKROOT = macosx;
+ 				WRAPPER_EXTENSION = prefPane;
++				SKIP_INSTALL = YES;
```

Comment:
At least in this fork, it normally, installs to `~/Library/PreferencePanes/`. I had to put this in to avoid that from happening. This allowed me to keep the existing `destroot` `file copy` code mostly untouched.